observation_portal/accounts/management/commands/generateschema_mocked.py

```diff
@@ -0,0 +1,21 @@
+import os
+import json
+
+from rest_framework.management.commands.generateschema import Command as GenerateSchemaCommand
+from django.conf import settings
+import responses
+
+CONFIGDB_DOC_FILE = os.path.join(settings.BASE_DIR, 'observation_portal/common/test_data/doc_configdb.json')
```

Comment:
Right now this configdb file is a direct copy of our test configdb file - it may be worth slimming it down a bit and removing some LCO-specific names. Can do that if we think it's a good idea.